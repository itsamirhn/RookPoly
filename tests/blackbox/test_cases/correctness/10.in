5
00001
00111
01011
01101
11110
