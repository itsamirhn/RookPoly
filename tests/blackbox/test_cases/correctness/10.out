4X^5 + 26X^4 + 49X^3 + 37X^2 + 11X + 1
